share/static/html/page.html

```diff
@@ -107,7 +107,7 @@ <h1 class="mdl-typography--display-2">
   {% block footer %}
   <footer class="mdl-mini-footer everware-footer">
       <div class="mdl-mini-footer__left-section">
-        Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
+        v0.5.0 Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
```

Comment:
there are not so many pages rendered, so passing `version` argument should not be much hassle. New page would be pick this field by nature of copy-n-paste.